vehicle/mkv/software/air_control/tests/test_initial_checks.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+import time
+from formula.projects.microhitl import Values as PinValue
+
+
+def reset(iocontroller, pins):
+    iocontroller.write_pin(pins["RESET"][0], PinValue.LOW)
+    time.sleep(0.5)
+    iocontroller.write_pin(pins["RESET"][0], PinValue.HIGH)
+
+
+def test_bms_can_timeout(canbus, iocontroller, pins):
+    """
+    Fault during init if BMS message takes longer than 1 second to receive
+    """
+    reset(iocontroller, pins)
+    assert canbus.get_state("air_fault") == "NONE"
+    time.sleep(1.1)
+    assert canbus.get_state("air_state") == "FAULT"
+    assert canbus.get_state("air_fault") == "CAN_BMS_TIMEOUT"
+
+
+def test_bms_voltage_low(canbus, iocontroller, pins):
+    """
+    Fault during init if BMS voltage is less than 200V
+    """
+    canbus.set_state("pack_voltage", 100)
+    canbus.set_periodic("bms_core", 0.3)
+
+    reset(iocontroller, pins)
+    assert canbus.get_state("air_fault") == "NONE"
+    time.sleep(1)
+    assert canbus.get_state("air_state") == "FAULT"
+    assert canbus.get_state("air_fault") == "BMS_VOLTAGE"
```

Comment:
I think all the "air_fault" states should be made to match the values for "air_fault" in the air.yml file. For example, here, the air_fault is "BMS_VOLTAGE", but the values in the yaml file all follow the format of "FAULT_BMS_VOLTAGE".

Reply:
This is a weird thing... I couldn't figure out why it was happening, but this was the only way to get the tests to pass. I think possibly it has to do with something that happens in the python CAN library we are using.